images/ipfailover/keepalived/conf/settings.sh

```diff
@@ -38,11 +38,35 @@ HA_REPLICA_COUNT=${OPENSHIFT_HA_REPLICA_COUNT:-"1"}
 
 #  Offset value to use to set the virtual router ids. Using different offset
 #  values allows multiple ipfailover configurations to exist within the
-#  same cluster.
+#  same cluster. Range 1..255
 #     HA_VRRP_ID_OFFSET=30
 #
 HA_VRRP_ID_OFFSET=${OPENSHIFT_HA_VRRP_ID_OFFSET:-"0"}
 
+# When the DC supplies an (non null) iptables chain
+# (OPENSHIFT_HA_IPTABLES_CHAIN) make sure the rule to pass keepalived
+# multicast (224.0.0.18) traffic is in the table.
+HA_IPTABLES_CHAIN=${OPENSHIFT_HA_IPTABLES_CHAIN:-""}
```

Comment:
@stevekuznetsov I shifted OPENSHIFT_HA_IPTABLES_CHAIN to here to be more consistent. 
I didn't create this file and am just adding a few fields. I am not sure how far to go with this.

This is work in progress. Thanks for your comments and I will modify the code as suggested. I am not sure how far I should go beyond adding this feature. Three is discussion on converting this to a go program and using the go template to generate the file. However, that is future work (if we decide to do it.)

The scripting here generates the config file for keepalived.